tests/template_backends/test_dummy.py

```diff
@@ -2,9 +2,13 @@
 
 from __future__ import unicode_literals
 
+import re
+
 from django.forms import CharField, Form, Media
 from django.http import HttpRequest
-from django.middleware.csrf import CsrfViewMiddleware, get_token
+from django.middleware.csrf import (
+    CsrfViewMiddleware, _compare_padded_tokens as equivalent_tokens, get_token,
```

Comment:
I guess it should be `_compare_salted_tokens` (rather than "padded") to fix build errors.